backend/open_webui/routers/users.py

```
@@ -333,11 +352,22 @@ async def update_user_by_id(
     # Prevent modification of the primary admin user by other admins
     try:
         first_user = Users.get_first_user()
-        if first_user and user_id == first_user.id and session_user.id != user_id:
-            raise HTTPException(
-                status_code=status.HTTP_403_FORBIDDEN,
-                detail=ERROR_MESSAGES.ACTION_PROHIBITED,
-            )
+        if first_user:
+            if user_id == first_user.id:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)